### ISSUE TITLE
Player's pals not showing in v0.4.8(latest) but do on previous versions.

### ISSUE THREAD

**Issue by ghost** [EternalWraith/PalEdit#53]
Not sure if anyone else has experienced this, but A LOT of my pals aren't showing up in the sidebar under my name. v0.4.8 shows only about 20 odd pals, when my character has a few boxes full.

To confirm this, I have a Wixen nicknamed "APPEAR IN PALEDIT" to differentiate it easily from others and identify it in the list of pals. 

You can see it appears in the older version, but not in the new version (name blanked bc its personally identifiable, same with one of the nicknames). Not under my name or any of the other players.

Both are using the exact same save file which is my latest save. What's even more strange is some of the pals appearing on the v0.4.8 window don't currently exist, or I've long changed their name from what it's appearing as in the window.

![image](https://github.com/EternalWraith/PalEdit/assets/158117191/1b6ddd1d-c716-45c9-9a11-148f0f9b6d17)

I will test this again on my other computer in case I'm either going crazy or doing something wrong.

**Comment by tsaij2** [comment]
Yeah I noticed what happened here. The Pal it listed under the character who CAPTURED it as opposed to the character who currently has it in their inventory. Check the other character and you'll be able to find it!

**Comment by paltester01** [comment]
I've tried it again on my laptop and I'm still getting the same results. 

> Yeah I noticed what happened here. The Pal it listed under the character who CAPTURED it as opposed to the character who currently has it in their inventory. Check the other character and you'll be able to find it!

This doesn't make sense because it for one does not explain how Pals that I don't have anymore or Pals who's named I have changed are having their old nicknames/appearing under my name in the list. 

>The Pal it listed under the character who CAPTURED it as opposed to the character who currently has it in their inventory. Check the other character and you'll be able to find it!

I checked all characters and couldn't find the Wixen like in the second image on v0.4.6, any of them, I caught a few of them and then renamed one of them to "APPEAR IN PALEDIT" to explain the issue.

**Comment by averon420** [comment]
@paltester01 can you please join discord. https://discord.gg/4srbJH6r 

As for your issue, removed pals from game (sold etc) are still in the level.sav (for what ever reasons) we currently can't find a solution to identfy those.
